OpenShift/helm/nexus-iq/Chart.yaml

```diff
@@ -1,9 +1,25 @@
-apiVersion: v1
-appVersion: 1.84.0
-description: A Helm chart for Kubernetes
-name: iqserver
-version: 0.1.0
+apiVersion: v2
+name: nexus-iq-server
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 90.0.0
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application.
+appVersion: 1.90.0
+
 description: Sonatype Nexus IQ Server continuously monitors your entire software supply chain
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
```

Comment:
i like the addition of the docs. 👍

Reply:
came from the base templates created by "helm create <project>" so I wanted to preserve that